### PR TITLE
Create advanced playground; simplify basic playground

### PR DESCRIPTION
**Pull request by rachel-fenichel**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Part of https://github.com/google/blockly/issues/3882

### Proposed Changes

Create advanced playground:
- Use the @blockly/dev-tools package to import the debug renderer, simple toolbox, category toolbox, and gui controls.
- Remove lots of checkboxes and dropdowns in favor of gui controls.
- Modify airstrike to randomly select from all blocks instead of all blocks in the toolbox.
  - Open to discussion

Update the standard playground:
 - Remove various controls + logic:
  - Debug renderer
  - Renderer picker
  - Theme picker
  - Accessibility checkbox

### Reason for Changes

Use published dev tools where possible.
Simplify standard playground while continuing to have a functional playground that works w/no internet access, installation, etc.

### Test Coverage

Opened both playgrounds and clicked around.

### Documentation
None

### Additional Information

There's more that can be done. I'm tracking some ideas in #3882.

### CODE REVIEW

File: tests/playgrounds/advanced_playground.html [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+<script src="../../blocks/text.js"></script>
+<script src="../../blocks/lists.js"></script>
+<script src="../../blocks/colour.js"></script>
+<script src="../../blocks/variables.js"></script>
+<script src="../../blocks/variables_dynamic.js"></script>
+<script src="../../blocks/procedures.js"></script>
+<script src="../blocks/test_blocks.js"></script>
+<script src="../themes/test_themes.js"></script>
+<script src="./screenshot.js"></script>
+<script src="https://unpkg.com/@blockly/dev-tools@0.20200313.7/dist/index.js"></script>
```
Comment (samelhusseini) [inline tests/playgrounds/advanced_playground.html:70]: If we leave out the version, we'll just get latest: 
``
https://unpkg.com/@blockly/dev-tools/dist/index.js
``

File: tests/playgrounds/advanced_playground.html [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+<script src="https://unpkg.com/@blockly/theme-modern@1.20200417.5/dist/index.js"></script>
```
Comment (samelhusseini) [inline tests/playgrounds/advanced_playground.html:71]: Same here, leave out version.

File: tests/playgrounds/advanced_playground.html [hunk 3]
```diff
@@ -0,0 +1,548 @@
...
+
+<script>
+// // Custom requires for the playground.
```
Comment (samelhusseini) [inline tests/playgrounds/advanced_playground.html:74]: nit: remove extra comment tag.

File: tests/playgrounds/advanced_playground.html [hunk 4]
```diff
@@ -0,0 +1,548 @@
...
+
+function start() {
+  setBackgroundColour();
+
+  // Parse the URL arguments.
+  var toolbox = getToolboxElement();
+  var autoimport = !!location.search.match(/autoimport=([^&]+)/);
+  var match = location.search.match(/renderer=([^&]+)/);
+
+  var options = {
```
Comment (samelhusseini) [inline tests/playgrounds/advanced_playground.html:95]: rename to `defaultOptions`.

File: tests/playgrounds/advanced_playground.html [hunk 5]
```diff
@@ -0,0 +1,548 @@
...
+            maxScale: 4,
+            minScale: 0.25,
+            scaleSpeed: 1.1
+          }
+  };
+
+  // Create main workspace.
+  addGUIControls((options) => {
+    workspace = Blockly.inject('blocklyDiv', options);
+    workspace.configureContextMenu = configureContextMenu;
```
Comment (samelhusseini) [inline tests/playgrounds/advanced_playground.html:136]: I would be in favour of pulling this out into a "createWorkspace" function along side the rest of the functions in here.

Comment (rachel-fenichel) [reply tests/playgrounds/advanced_playground.html:156]: Done

**Pull request merged**